### PR TITLE
[hailctl] Move hailctl inside hail/python

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: hail/Makefile [hunk 1]
```diff
@@ -77,17 +77,17 @@ DEV_SALT := $(shell whoami)-dev
 endif
 
 CLOUD_LOC = gs://hail-common/hailctl/dataproc/$(DEV_SALT)/$(HAIL_PIP_VERSION)/$(SHORT_REVISION)/$(FILENAME)
-RESOURCES_TO_COPY = $(shell ls ../hailctl/python/hailctl/dataproc/resources)
-../hailctl/python/hailctl/deploy.yaml:
-	rm -f ../hailctl/python/hailctl/deploy.yaml
-	echo "dataproc:" >> ../hailctl/python/hailctl/deploy.yaml
+RESOURCES_TO_COPY = $(shell ls python/hailctl/dataproc/resources)
+python/hailctl/deploy.yaml:
+	rm -f python/hailctl/deploy.yaml
```
Comment (jigold) [inline hail/Makefile]: Do you need this line? Isn't this what .PHONY does?

Comment (tpoterba) [reply hail/Makefile]: not really. But I didn't change this line in the PR -- just the paths these things point to, so I think the change is out of scope here

Comment (tpoterba) [reply hail/Makefile]: I think in a previous version I was appending to the file in various places, and wanted to be sure I was starting fresh

File: hail/python/hailctl/dataproc/resources/init_notebook.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```
Comment (tpoterba) [inline hail/python/hailctl/dataproc/resources/init_notebook.py]: oops, I bundled these changes accidentally. The init script is still not functional (did the one you PRed actually work to generate clusters?) but is closer.

Comment (jigold) [reply hail/python/hailctl/dataproc/resources/init_notebook.py]: I didn't test it at all. All I did was startup dataproc without the original init script and see where everything was located.

Comment (tpoterba) [reply hail/python/hailctl/dataproc/resources/init_notebook.py]: OK, sure. I'll get this stuff working next.

**Pull request merged**